src/integration-test/resources/DigitalInvoiceIT/__files/test2_internalServerErrorFromDigitalMailSender/request.json

```diff
@@ -0,0 +1,23 @@
+{
+  "party": {
```

Comment:
Minor issue:

Use same indentation-length on all json-files?
(some files use indentation-length of 2 spaces, other 4 spaces)